casepro/cases/forms.py

```diff
@@ -7,36 +7,37 @@
 from .models import Partner
 
 
-class PartnerUpdateForm(forms.ModelForm):
+class BasePartnerForm(forms.ModelForm):
+    description = forms.CharField(label=_("Description"), max_length=255, required=False, widget=forms.Textarea)
```

Comment:
Moving the common description/labels functionality into a base form class to avoid duplication